File: app/src/main/java/in/woloo/www/trendingblog/epoxy/NoBlogsModel.java
package in.woloo.www.trendingblog.epoxy;

import android.view.View;
import android.widget.RelativeLayout;
import android.widget.TextView;

import androidx.appcompat.widget.AppCompatImageView;

import com.airbnb.epoxy.EpoxyModelClass;
import com.airbnb.epoxy.EpoxyModelWithHolder;

import butterknife.BindView;
import in.woloo.www.R;
import in.woloo.www.base.BaseEpoxyHolder;

@EpoxyModelClass(layout = R.layout.model_no_blogs)
public abstract class NoBlogsModel extends EpoxyModelWithHolder<NoBlogsModel.Holder> {

    static class Holder extends BaseEpoxyHolder {
        @BindView(R.id.tvNoBlogs)
        TextView tvNoBlogs;
    }
}

File: app/src/main/java/in/woloo/www/more/trendingblog/epoxy/NoBlogsModel.kt
package `in`.woloo.www.more.trendingblog.epoxy

import android.widget.TextView
import butterknife.BindView
import com.airbnb.epoxy.EpoxyModelClass
import com.airbnb.epoxy.EpoxyModelWithHolder
import `in`.woloo.www.R
import `in`.woloo.www.application_kotlin.base.BaseEpoxyHolder

@EpoxyModelClass(layout = R.layout.model_no_blogs)
abstract class NoBlogsModel : EpoxyModelWithHolder<NoBlogsModel.Holder?>() {

    class Holder : BaseEpoxyHolder() {
        @JvmField
        @BindView(R.id.tvNoBlogs)
        var tvNoBlogs: TextView? = null
    }
}
